src/txdb.cpp

```diff
@@ -5,6 +5,7 @@
 
 #include "txdb.h"
 
+#include "addressindex.h"
```

Comment:
I wonder if this include should go under a conditionally compiled section since it is used by the Address Indexing only.

```suggestion
#ifdef ENABLE_ADDRESS_INDEXING
#include "addressindex.h"
#endif // ENABLE_ADDRESS_INDEXING
```

Reply:
Removed because already included by txdb.h